go.mod

```diff
@@ -31,4 +31,4 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-go 1.15
+go 1.16
```

Comment:
How about `examples/nft-d/go.mod`?

Reply:
README.md needs to also be updated,

Reply:
Finally, I was able to push changes to remote, so fixed.

Reply:
I'm not sure, is it enought to update gomod of `nft-d` contract or we need to update the `neo-go`'s gomod also?

Reply:
But how can we squash them? The perpose of the update is to use `neo-go` dependency of `1.16` version everywhere, so it should be a separate commit.

Reply:
OK.

Reply:
Fixed, ready for review.